webview-ui/src/components/chat/CreditLimitError.tsx

```diff
@@ -19,6 +22,12 @@ const CreditLimitError: React.FC<CreditLimitErrorProps> = ({
 	message = "You have run out of credit.",
 	buyCreditsUrl = "https://app.cline.bot/dashboard/account?tab=credits&redirect=true",
 }) => {
+	const { uriScheme } = useExtensionState()
+
+	const callbackUrl = `${uriScheme || "vscode"}://saoudrizwan.claude-dev`
+	const fullPurchaseUrl = new URL(buyCreditsUrl)
+	fullPurchaseUrl.searchParams.set("callback_url", callbackUrl)
```

Comment:
```suggestion
	fullPurchaseUrl.searchParams.set("tab", credits)
	fullPurchaseUrl.searchParams.set("redirect", true)
	fullPurchaseUrl.searchParams.set("callback_url", callbackUrl)
```
Not sure if we should set these search params in case it was missing?

Reply:
we changed the way it works - no need for tab anymore.